server/control-panel.md

```diff
@@ -1,69 +1,39 @@
 # Control Panels
 
-## cPanel
+This document describes how to install WordPress® in the cPanel & WHM and Plesk control panels using WP Toolkit.
```

Comment:
This document should be generic for multiple control panels and this sentence should be under the cPanel and Plesk headlines